src/components/MultiStepForm/MultiStepForm.tsx

```diff
@@ -79,21 +111,34 @@ const FormContainer: FC<MultiStepFormProps> = ({ formTemplate, formData, isLoadi
             {subHeader && <h4 className={styles.subHeader}>{subHeader}</h4>}
           </div>
           {formComponents.map(
-            ({ componentType, componentData, formComponentLink, classNameSuffix }, index) => (
-              <div
-                className={`${styles.formComponent} ${
-                  classNameSuffix ? styles[classNameSuffix] : ''
-                }`}
-                key={index}
-              >
-                {createFormComponent(componentType, formData, componentData, setPageNumber)}
-                {formComponentLink && (
-                  <div className={styles.link}>
-                    <ExternalLink {...formComponentLink} />
-                  </div>
-                )}
-              </div>
-            )
+            ({ componentType, componentData, formComponentLink, classNameSuffix }, index) => {
+              let errorMessage;
+              const { formMeta: { field = '' } = {} } = componentData as CommonInputProps;
+              if (field in formErrors) {
```

Comment:
I would just remove line 155 and do `const errorMessage = field in formErrors && formErrors[field]`

Reply:
thanks, resolved. I've used a ternary which sets `errorMessage` to an empty string (a falsy value, thus providing the same behaviour as previous logic) if the first condition is not met.